OrchardCore.Commerce.Tests/CurrencyTests.cs

```diff
@@ -18,7 +18,7 @@ public static CurrencyTheoryData TestData
                 { CanadianDollar, 1234.56m, "$1,234.56" },
                 { SwissFranc, 1234.56m, "CHF 1’234.56" },
                 { ChineseYuan, 1234.56m, "¥1,234.56" },
-                { new Currency("My FOO", "My FOO", "f", "FOO"), 1234.56m, "(FOO) 1,234.56" }
+                { new Currency("My FOO", "My FOO", "f", "FOO"), 1234.56m, $"(FOO) {1234.56m.ToString("N")}" }
```

Comment:
Not sure, thinking about it.

Reply:
A null culture normally is not appropriate so it should fallback to InvariantCulture formatting which allows usage of comma for thousands separator.